src/patternfly/base/patternfly-common.scss

```
@@ -64,13 +64,69 @@
   --pf-t--global--duration--200: 1ms !important;
   --pf-t--global--duration--100: 1ms !important;
   --pf-t--global--duration--50: 1ms !important;
+  --#{$pf-global}--pulse-pulsing--Animation--Duration: 0 !important;
 
   // stylelink-enable declaration-no-important
 }
 
+.#{$pf-prefix}m-gradient-border {
```

I pinged @lboehling about this and since this is something that initiates an AI chat/interaction, she recommended "ai-trigger" or "ai-indicator".

```suggestion
.#{$pf-prefix}m-ai-trigger {
```